typescript/ccip-server/src/config.ts

```diff
@@ -3,21 +3,15 @@ import dotenvFlow from 'dotenv-flow';
 dotenvFlow.config();
 
 const RPC_ADDRESS = process.env.RPC_ADDRESS as string;
-const LIGHT_CLIENT_ADDR = process.env.LIGHT_CLIENT_ADDR as string;
```

Comment:
I didn't realize that we have this already in the monorepo. I think we should have an opinion on whether they should be living in the monorepo. IMO at minimum it should be modular so that the default is not SP1?

Reply:
Can we do this in a separate PR considering the current scope? Seems like community-ism repo is not ready yet and needs to be organized (looks mostly to be used for contracts right now)